helm-charts/basehub/values.yaml

```diff
@@ -445,10 +445,33 @@ jupyterhub:
         args:
           - -c
           - |
-            ls -lhd /srv/repo;
-            while true; do git fetch origin;
-            git reset --hard origin/$(GIT_REPO_BRANCH);
-            sleep 5m; done
+            handle_sigterm() {
+                echo "SIGTERM received, terminating...";
+                exit;
+            }
+            trap handle_sigterm SIGTERM
```

Comment:
Adding this is the key thing for this PR, but it also had to be combined with breaking apart sleep 5m into 500 sleep 1s. The rest of the changes is just about providing some misc info of relevance.